src/python/pants/backend/python/rules/pex.py

```diff
@@ -4,23 +4,30 @@
 from pants.backend.python.rules.download_pex_bin import DownloadedPexBin
```

Comment:
I think we want to keep the name something like `create_pex.py` instead of `pex.py`. In general our rules often have verb-based names.

Reply:
Yeah defining all types of pex creation rules in this file makes sense I think, it's what I plan to do in future commits